post-processor/vagrant-cloud/client.go

```diff
@@ -102,33 +102,33 @@ func (v VagrantCloudClient) Delete(path string) (*http.Response, error) {
 }
 
 func (v VagrantCloudClient) Upload(path string, url string) (*http.Response, error) {
-	file, err := os.Open(path)
```

Comment:
These changes don't seem related to the progress bar change.

Reply:
yea. you're right. i'll find which commit added this an remove it.

you might want to fix this tho as it's a file handle leak.